controllers/storagecluster/rookoperatorhostnetwork.go

```diff
@@ -55,6 +56,7 @@ func (obj *rookCephCsvHostNetwork) ensureCreated(r *StorageClusterReconciler, in
 			}
 			// Enable host network for rook-ceph-operator
 			deployment.Spec.Template.Spec.HostNetwork = shouldUseHostNetworking(instance)
+			deployment.Spec.Template.Spec.DNSPolicy = corev1.DNSClusterFirstWithHostNet
```

Comment:
@rohan47 Can you please explain whey this is now set everytime?

Reply:
This is required for DNS resolution for openshift service when hostNetwork is set to true